caracal/workers/inspect_worker.py

```diff
@@ -586,6 +586,6 @@ def worker(pipeline, recipe, config):
                         "step":  f"plot-{axes}-{iobs}-{ftype[0]}",
                         "label": label,
                         **plot_axes[axes]})
-
-                    globals()[plotter](pipeline, recipe, plot_args,
-                                       extras=None)
+                    if plotter and plotter != "none":
+                        globals()[plotter](pipeline, recipe, plot_args,
+                                           extras=None)
```

Comment:
This works, but why don't you move the if outside of the two for-loops to avoid looping uselessly?

Reply:
Done